src/components/reservationForm/ReservationForm.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import Select from 'react-select';
+import Modal from '../modal/Modal';
+import './reservationForm.css';
+
+const cities = [
+  { value: 'New York', label: 'New York' },
+  { value: 'Los Angeles', label: 'Los Angeles' },
+  { value: 'Oregon', label: 'Oregon' },
+  { value: 'Bradenton Beach', label: 'Bradenton Beach' },
+  { value: 'Charlottetown', label: 'Charlottetown' },
+  { value: 'Bangkog', label: 'Bangkog' },
+  { value: 'Beijing', label: 'Beijing' },
+];
+
+const ReservationForm = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [selectedCity, setSelectedCity] = useState('');
+
+  const handleModalOpen = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleModalClose = () => {
+    setIsModalOpen(false);
+  };
+
+  return (
+    <div className="container">
+      <div className="header">
+        <div className="nav">
+          <div className="nav-icon">
+            <i className="fas fa-bars" />
+          </div>
+          <div className="search-icon">
+            <i className="fas fa-search" />
+          </div>
+        </div>
+        <div className="title">
+          <h3>BOOK A TEST DRIVE WITH WHEEL-WIZARD</h3>
+          <hr className="divider" />
+          <p style={{ color: '#fff' }}>
+            Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ducimus, blanditiis. lorem10
+          </p>
+        </div>
+        <div className="buttons" style={{ display: 'flex', alignItems: 'center', marginTop: '2rem' }}>
```

Comment:
Definitely got to remove this lorem ipsum and put something else, 😅 
Also, in general, some work can be done to improve the design. The select in particular looks a bit wonky.

![image](https://user-images.githubusercontent.com/31193823/229302926-428f59fc-74c5-4802-9ac5-84f800d1a50b.png)